.github/workflows/Docker Test.yml

```diff
@@ -63,17 +63,28 @@ jobs:
       matrix:
         StandardVersion: ${{ fromJson(needs.FindStandardAndExtensionVersions.outputs.StandardVersions) }}
         ExtensionVersion: ${{ fromJson(needs.FindStandardAndExtensionVersions.outputs.ExtensionVersions) }}
+        dockerfile:
+          [
+            { name: "ods-api-db-admin", path: "alpine/pgsql" },
+            { name: "ods-api-db-ods-minimal", path: "alpine/pgsql" },
+            { name: "ods-api-db-ods-sandbox", path: "alpine/pgsql" },
+            { name: "ods-api-swaggerui", path: "alpine" },
+            { name: "ods-api-web-api", path: "alpine/pgsql" },
+            { name: "ods-api-web-api", path: "alpine/mssql" },            
+            { name: "ods-api-web-sandbox-admin", path: "alpine/pgsql" },
+            { name: "ods-api-web-sandbox-admin", path: "alpine/mssql" }            
+          ]        
     name: Docker Build (Standard ${{ matrix.StandardVersion }} Extension ${{ matrix.ExtensionVersion }})
```

Comment:
We should add the image name in the job name, something like ImageName (Std, Ext) to make it easy finding which image we are looking at and not a list of 'Docker Build ...'
<img width="620" alt="image" src="https://github.com/Ed-Fi-Alliance-OSS/Ed-Fi-ODS-Implementation/assets/43448209/fd6be146-36f1-4c3d-98af-e19d0acd9be3">

Reply:
Updated . Thanks